### ISSUE TITLE
npm warnings when installing

### ISSUE THREAD

**Issue by yuhui** [adobe/reactor-sandbox#50]
When running `npx @adobe/reactor-sandbox` for the first time, a list of warnings, including some about deprecated dependencies, are listed.

### Expected Behaviour
No warnings are listed.

### Actual Behaviour
Warnings appear.

### Reproduce Scenario (including but not limited to)

macOS Big Sur 11.0.1
node 15.2.1
npm 7.0.8

`node` and `npm` installed with `brew`.

#### Steps to Reproduce

1. Run `npx @adobe/reactor-sandbox`

#### Sample Code that illustrates the problem

```
yuhui@computer launch-extension-folder % npx @adobe/reactor-sandbox
Need to install the following packages:
  @adobe/reactor-sandbox
Ok to proceed? (y) y
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: react@17.0.1
npm WARN node_modules/react
npm WARN   react@"^17.0.1" from @adobe/reactor-sandbox@11.0.2
npm WARN   node_modules/@adobe/reactor-sandbox
npm WARN     @adobe/reactor-sandbox@"*" from the root project
npm WARN   6 more (@react-spectrum/icon, @react-spectrum/provider, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer react@">=15.5 <=16.x" from react-codemirror2@7.2.1
npm WARN node_modules/react-codemirror2
npm WARN   react-codemirror2@"^7.2.1" from @adobe/reactor-sandbox@11.0.2
npm WARN   node_modules/@adobe/reactor-sandbox
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: react@17.0.1
npm WARN node_modules/react
npm WARN   react@"^17.0.1" from @adobe/reactor-sandbox@11.0.2
npm WARN   node_modules/@adobe/reactor-sandbox
npm WARN     @adobe/reactor-sandbox@"*" from the root project
npm WARN   6 more (@react-spectrum/icon, @react-spectrum/provider, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer react@"^15.5.4 || ^16.0.0" from react-router-last-location@2.0.1
npm WARN node_modules/react-router-last-location
npm WARN   react-router-last-location@"^2.0.1" from @adobe/reactor-sandbox@11.0.2
npm WARN   node_modules/@adobe/reactor-sandbox
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: react-dom@17.0.1
npm WARN node_modules/react-dom
npm WARN   react-dom@"^17.0.1" from @adobe/reactor-sandbox@11.0.2
npm WARN   node_modules/@adobe/reactor-sandbox
npm WARN     @adobe/reactor-sandbox@"*" from the root project
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer react-dom@"^15.5.4 || ^16.0.0" from react-router-last-location@2.0.1
npm WARN node_modules/react-router-last-location
npm WARN   react-router-last-location@"^2.0.1" from @adobe/reactor-sandbox@11.0.2
npm WARN   node_modules/@adobe/reactor-sandbox
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: react@17.0.1
npm WARN node_modules/react
npm WARN   react@"^17.0.1" from @adobe/reactor-sandbox@11.0.2
npm WARN   node_modules/@adobe/reactor-sandbox
npm WARN     @adobe/reactor-sandbox@"*" from the root project
npm WARN   6 more (@react-spectrum/icon, @react-spectrum/provider, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer react@"^16.8.0" from @adobe/react-spectrum-ui@1.0.1
npm WARN node_modules/@spectrum-icons/illustrations/node_modules/@adobe/react-spectrum-ui
npm WARN   @adobe/react-spectrum-ui@"^1.0.0" from @spectrum-icons/illustrations@3.2.1
npm WARN   node_modules/@spectrum-icons/illustrations
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: react@17.0.1
npm WARN node_modules/react
npm WARN   react@"^17.0.1" from @adobe/reactor-sandbox@11.0.2
npm WARN   node_modules/@adobe/reactor-sandbox
npm WARN     @adobe/reactor-sandbox@"*" from the root project
npm WARN   6 more (@react-spectrum/icon, @react-spectrum/provider, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer react@"^16.14.0" from react-dom@16.14.0
npm WARN node_modules/@spectrum-icons/illustrations/node_modules/react-dom
npm WARN   peer react-dom@"^16.8.0" from @adobe/react-spectrum-ui@1.0.1
npm WARN   node_modules/@spectrum-icons/illustrations/node_modules/@adobe/react-spectrum-ui
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: react@17.0.1
npm WARN node_modules/react
npm WARN   react@"^17.0.1" from @adobe/reactor-sandbox@11.0.2
npm WARN   node_modules/@adobe/reactor-sandbox
npm WARN     @adobe/reactor-sandbox@"*" from the root project
npm WARN   6 more (@react-spectrum/icon, @react-spectrum/provider, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer react@"^16.8.0" from @adobe/react-spectrum-ui@1.0.1
npm WARN node_modules/@spectrum-icons/ui/node_modules/@adobe/react-spectrum-ui
npm WARN   @adobe/react-spectrum-ui@"^1.0.0" from @spectrum-icons/ui@3.2.0
npm WARN   node_modules/@spectrum-icons/ui
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: react@17.0.1
npm WARN node_modules/react
npm WARN   react@"^17.0.1" from @adobe/reactor-sandbox@11.0.2
npm WARN   node_modules/@adobe/reactor-sandbox
npm WARN     @adobe/reactor-sandbox@"*" from the root project
npm WARN   6 more (@react-spectrum/icon, @react-spectrum/provider, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer react@"^16.14.0" from react-dom@16.14.0
npm WARN node_modules/@spectrum-icons/ui/node_modules/react-dom
npm WARN   peer react-dom@"^16.8.0" from @adobe/react-spectrum-ui@1.0.1
npm WARN   node_modules/@spectrum-icons/ui/node_modules/@adobe/react-spectrum-ui
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: react@17.0.1
npm WARN node_modules/react
npm WARN   react@"^17.0.1" from @adobe/reactor-sandbox@11.0.2
npm WARN   node_modules/@adobe/reactor-sandbox
npm WARN     @adobe/reactor-sandbox@"*" from the root project
npm WARN   6 more (@react-spectrum/icon, @react-spectrum/provider, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer react@"^16.8.0" from @adobe/react-spectrum-workflow@1.0.1
npm WARN node_modules/@spectrum-icons/workflow/node_modules/@adobe/react-spectrum-workflow
npm WARN   @adobe/react-spectrum-workflow@"^1.0.0" from @spectrum-icons/workflow@3.2.0
npm WARN   node_modules/@spectrum-icons/workflow
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: react@17.0.1
npm WARN node_modules/react
npm WARN   react@"^17.0.1" from @adobe/reactor-sandbox@11.0.2
npm WARN   node_modules/@adobe/reactor-sandbox
npm WARN     @adobe/reactor-sandbox@"*" from the root project
npm WARN   6 more (@react-spectrum/icon, @react-spectrum/provider, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer react@"^16.14.0" from react-dom@16.14.0
npm WARN node_modules/@spectrum-icons/workflow/node_modules/react-dom
npm WARN   peer react-dom@"^16.8.0" from @adobe/react-spectrum-workflow@1.0.1
npm WARN   node_modules/@spectrum-icons/workflow/node_modules/@adobe/react-spectrum-workflow
npm WARN deprecated request-promise-native@1.0.9: request-promise-native has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
npm WARN deprecated chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm WARN deprecated @hapi/topo@3.1.6: This version has been deprecated and is no longer supported or maintained
npm WARN deprecated @hapi/bourne@1.3.2: This version has been deprecated and is no longer supported or maintained
npm WARN deprecated rollup-plugin-babel@4.4.0: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-babel.
npm WARN deprecated fsevents@2.1.3: Please update to v 2.2.x
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated @hapi/hoek@8.5.1: This version has been deprecated and is no longer supported or maintained
npm WARN deprecated @hapi/address@2.1.4: Moved to 'npm install @sideway/address'
npm WARN deprecated @hapi/joi@15.1.1: Switch to 'npm install joi'
npm WARN deprecated core-js@2.6.12: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
```

**Comment by yuhui** [comment]
Updated set of warnings

Environment:
node 15.8.0
npm 7.5.4

```
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: react@17.0.1
npm WARN node_modules/react
npm WARN   react@"^17.0.1" from @adobe/reactor-sandbox@11.1.2
npm WARN   node_modules/@adobe/reactor-sandbox
npm WARN     @adobe/reactor-sandbox@"*" from the root project
npm WARN   7 more (@adobe/reactor-extension-core-edge, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer react@">=15.5 <=16.x" from react-codemirror2@7.2.1
npm WARN node_modules/react-codemirror2
npm WARN   react-codemirror2@"^7.2.1" from @adobe/reactor-sandbox@11.1.2
npm WARN   node_modules/@adobe/reactor-sandbox
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: react@17.0.1
npm WARN node_modules/react
npm WARN   react@"^17.0.1" from @adobe/reactor-sandbox@11.1.2
npm WARN   node_modules/@adobe/reactor-sandbox
npm WARN     @adobe/reactor-sandbox@"*" from the root project
npm WARN   7 more (@adobe/reactor-extension-core-edge, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer react@"^15.5.4 || ^16.0.0" from react-router-last-location@2.0.1
npm WARN node_modules/react-router-last-location
npm WARN   react-router-last-location@"^2.0.1" from @adobe/reactor-sandbox@11.1.2
npm WARN   node_modules/@adobe/reactor-sandbox
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: react-dom@17.0.1
npm WARN node_modules/react-dom
npm WARN   react-dom@"^17.0.1" from @adobe/reactor-sandbox@11.1.2
npm WARN   node_modules/@adobe/reactor-sandbox
npm WARN     @adobe/reactor-sandbox@"*" from the root project
npm WARN   2 more (@adobe/reactor-extension-core-edge, react-json-view)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer react-dom@"^15.5.4 || ^16.0.0" from react-router-last-location@2.0.1
npm WARN node_modules/react-router-last-location
npm WARN   react-router-last-location@"^2.0.1" from @adobe/reactor-sandbox@11.1.2
npm WARN   node_modules/@adobe/reactor-sandbox
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: react@17.0.1
npm WARN node_modules/react
npm WARN   react@"^17.0.1" from @adobe/reactor-sandbox@11.1.2
npm WARN   node_modules/@adobe/reactor-sandbox
npm WARN     @adobe/reactor-sandbox@"*" from the root project
npm WARN   7 more (@adobe/reactor-extension-core-edge, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer react@"^16.8.0" from @react-spectrum/slider@3.0.1
npm WARN node_modules/@react-spectrum/slider
npm WARN   @react-spectrum/slider@"^3.0.1" from @adobe/react-spectrum@3.8.0
npm WARN   node_modules/@adobe/react-spectrum
npm WARN deprecated @hapi/topo@3.1.6: This version has been deprecated and is no longer supported or maintained
npm WARN deprecated @hapi/bourne@1.3.2: This version has been deprecated and is no longer supported or maintained
npm WARN deprecated rollup-plugin-babel@4.4.0: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-babel.
npm WARN deprecated @hapi/address@2.1.4: Moved to 'npm install @sideway/address'
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
npm WARN deprecated @hapi/hoek@8.5.1: This version has been deprecated and is no longer supported or maintained
npm WARN deprecated request-promise-native@1.0.9: request-promise-native has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated @hapi/joi@15.1.1: Switch to 'npm install joi'
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
npm WARN deprecated chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm WARN deprecated babel-eslint@10.1.0: babel-eslint is now @babel/eslint-parser. This package will no longer receive updates.
npm WARN deprecated core-js@2.6.12: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
```

**Comment by yuhui** [comment]
Updated set of warnings

Environment:
node 16.1.0
npm 7.11.2

```
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: react@17.0.2
npm WARN node_modules/react
npm WARN   react@"^17.0.1" from @adobe/reactor-sandbox@11.1.6
npm WARN   node_modules/@adobe/reactor-sandbox
npm WARN     dev @adobe/reactor-sandbox@"*" from the root project
npm WARN   7 more (@adobe/reactor-extension-core-edge, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer react@">=15.5 <=16.x" from react-codemirror2@7.2.1
npm WARN node_modules/react-codemirror2
npm WARN   react-codemirror2@"^7.2.1" from @adobe/reactor-sandbox@11.1.6
npm WARN   node_modules/@adobe/reactor-sandbox
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: react@17.0.2
npm WARN node_modules/react
npm WARN   react@"^17.0.1" from @adobe/reactor-sandbox@11.1.6
npm WARN   node_modules/@adobe/reactor-sandbox
npm WARN     dev @adobe/reactor-sandbox@"*" from the root project
npm WARN   7 more (@adobe/reactor-extension-core-edge, ...)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer react@"^15.5.4 || ^16.0.0" from react-router-last-location@2.0.1
npm WARN node_modules/react-router-last-location
npm WARN   react-router-last-location@"^2.0.1" from @adobe/reactor-sandbox@11.1.6
npm WARN   node_modules/@adobe/reactor-sandbox
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: react-dom@17.0.2
npm WARN node_modules/react-dom
npm WARN   react-dom@"^17.0.1" from @adobe/reactor-sandbox@11.1.6
npm WARN   node_modules/@adobe/reactor-sandbox
npm WARN     dev @adobe/reactor-sandbox@"*" from the root project
npm WARN   2 more (@adobe/reactor-extension-core-edge, react-json-view)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer react-dom@"^15.5.4 || ^16.0.0" from react-router-last-location@2.0.1
npm WARN node_modules/react-router-last-location
npm WARN   react-router-last-location@"^2.0.1" from @adobe/reactor-sandbox@11.1.6
npm WARN   node_modules/@adobe/reactor-sandbox
npm WARN deprecated @hapi/bourne@1.3.2: This version has been deprecated and is no longer supported or maintained
npm WARN deprecated @hapi/topo@3.1.6: This version has been deprecated and is no longer supported or maintained
npm WARN deprecated request-promise-native@1.0.9: request-promise-native has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
npm WARN deprecated chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
npm WARN deprecated chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm WARN deprecated babel-eslint@10.1.0: babel-eslint is now @babel/eslint-parser. This package will no longer receive updates.
npm WARN deprecated @hapi/address@2.1.4: Moved to 'npm install @sideway/address'
npm WARN deprecated rollup-plugin-babel@4.4.0: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-babel.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated @hapi/hoek@8.5.1: This version has been deprecated and is no longer supported or maintained
npm WARN deprecated intl-messageformat-parser@1.4.0: We've written a new parser that's 6x faster and is backwards compatible. Please use @formatjs/icu-messageformat-parser
npm WARN deprecated @hapi/joi@15.1.1: Switch to 'npm install joi'
npm WARN deprecated core-js@2.6.12: core-js@<3.3 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Please, upgrade your dependencies to the actual version of core-js.
```

**Comment by thebenrob** [comment]
@adobe export issue to Jira project PDCL

**Comment by dompuiu** [comment]
I fixed the warnings in 12.2.0. I don't see any warnings when I run `npx @adobe/reactor-sandbox`.
